atomistics/calculators/lammps.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from pylammpsmpi import LammpsASELibrary
+
+from atomistics.calculators.wrapper import task_evaluation
+from atomistics.calculators.lammps_library.calculator import (
+    _run_simulation,
+    lammps_input_template as _lammps_input_template,
+)
+
+if TYPE_CHECKING:
+    from ase import Atoms
+    from pandas import DataFrame
+    from pylammpsmpi import LammpsASELibrary
+
+
+@task_evaluation
+def evaluate_with_lammps_library(
+    structure: Atoms,
+    tasks: list[str],
+    potential_dataframe: DataFrame,
+    lmp: LammpsASELibrary,
+):
+    lmp = _run_simulation(structure, potential_dataframe, _lammps_input_template, lmp)
+    results = {}
+    if "calc_energy" in tasks:
+        results["energy"] = lmp.interactive_energy_pot_getter()
+    if "calc_forces" in tasks:
+        results["forces"] = lmp.interactive_forces_getter()
+    lmp.interactive_lib_command("clear")
+    return results
+
+
+def evaluate_with_lammps(
+    task_dict: dict[str, dict[str, Atoms]],
```

Comment:
I suppose this also needs a `@task_evaluation`, etc.

Reply:
Nope, because it is wrapping a function operating in the task dict space already.

Last night I was thinking I want a more explicit name for the decorator. It's just remapping from the single structure to task dict domain, with the intent of making the core function much easier for devs to write. But once that one's decorated the bells and whistles on top are already in the right domain.

Reply:
Ah, right. I missed that.  Ja, `task_evalution` feels a bit awkward, but not sure what's better.  Maybe `task_dict_evaluator` (corresponding to what I called `SimpleEvaluator` in the other PR)?

Reply:
> Maybe `task_dict_evaluator` (corresponding to what I called `SimpleEvaluator` in the other PR)?

Yes, I like that

Reply:
I went with `as_task_dict_evaluator` because I like it when decorators have verbs